src/components/candidate-form.js

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from "react";
+
+import styled from "styled-components";
+import axios from "axios";
+
+export default function CandidateForm(props) {
+  const { name, voteCount, rank, id } = props;
+
+  const voteCandidate = () => {
+    axios
+      .put(process.env.API_HOST + "/candidates/" + id + "/vote/")
+      .then(function (response) {
+        console.log(response);
+        alert(name + "님에게 투표완료.");
+      })
+      .catch(function (error) {
+        console.log(error);
+        alert("다시 시도해주세요.");
+      });
+  };
+
+  return (
+    <Wrapper>
+      <Rank>{rank}위:</Rank>
+      <Info>
+        {name}[{voteCount}표]
+      </Info>
+      <VoteButton
+        onClick={() => {
+          voteCandidate();
+        }}
```

Comment:
```suggestion
        onClick={voteCandidate}
```
함수명을 그대로 써주셔도 같습니다 ㅎㅎ